broadinstitute/ddp-study-server
# DSP-7581 export fixes for AT

This improves handling of composite questions, and multiple choice questions that allow specifying option details.  Now all values for those are given separate columns.  Likewise, if a participant gives multiple answers to a question (such as listing medications), each answer is given its own column (e.g. MEDICATIONS_1.NAME, MEDICATIONS_2.NAME, etc...)

TO TEST:
1.  Go to the AT study participant list in development
2.  Use the "Select all" button to select all columns
3.  Press the download button, and download using the default confiugration
4. For participant PKG8PA, confirm that the columns `MEDICAL_HISTORY.MEDICATION_CATEGORY.MEDICATION_NAME`, `MEDICAL_HISTORY.MEDICATION_CATEGORY.BEGAN_TAKING_AT_AGE` are filled out,  as are `MEDICAL_HISTORY.MEDICATION_CATEGORY_2.MEDICATION_NAME`, `MEDICAL_HISTORY.MEDICATION_CATEGORY_2.BEGAN_TAKING_AT_AGE`
5. For participant PKJD7H, confirm the column `MEDICAL_HISTORY_2.DETERMINED_OTHER_DETAILS` has a value of "Other diagnosis test"
6. generally confirm that the export has the correct number of participants, columns look correct, etc..
